lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -609,6 +675,21 @@ void SVExtractTestCodeImplPass::runOnOperation() {
   this->instanceGraph = &getAnalysis<circt::hw::InstanceGraph>();
 
   auto top = getOperation();
+
+  // It takes extra effort to inline modules which contains inner symbols
+  // referred through hierpaths or unknown operations since we have to update
+  // inner refs users globally. However we do want to inline modules which
+  // contain bound instances so create a set of inner refs used by non bind op
+  // in order to allow bind ops.
+  DenseSet<hw::InnerRefAttr> innerRefUsedByNonBindOp;
+  top.walk([&](hw::InnerRefUserOpInterface op) {
```

Comment:
I don't think the majority of the current inner-ref users in HW/related dialects implement this interface "yet".
Notably `BindInterfaceOp`, FWIW, but also in OM and such from a quick grep.  Might need to just walk everywhere for now, but probably this works "for now" as-is?

Reply:
Yep, good point. I'll just remove InnerRefUserOpInterface.